src/org/usfirst/frc/team2175/subsystem/drivetrain/DrivetrainSubsystem.java

```diff
@@ -67,14 +71,30 @@ public DrivetrainSubsystem() {
         analogGyro = new AnalogGyro(
                 wiringProperties.getDrivetrainAnalogGyroDeviceNumber());
 
+        // oldEncoder = new Encoder(sourceA, sourceB);
+
         // navXGyro = new AHRS(SPI.Port.kMXP);
         switchToPercentVbus();
+        // TODO Propertize
+        oldEncoder = new Encoder(5, 3, true);
+        oldEncoder.setDistancePerPulse(1);
+
     }
 
     public void arcadeDrive(final double moveValue, final double rotateValue) {
         robotDrive.arcadeDrive(-moveValue, rotateValue);
     }
 
+    public void straightArcadeDrive(final double moveValue,
+            final double rotateValue) {
+        if (!(Math.abs(getGyroAngle()) > .25)) {
```

Comment:
@NoahConnors Please just use `<=` instead of negating a greater than. Also, why does this method have a `rotateValue` parameter at all?